package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "dev": "next",
-    "build": "next build",
+    "build": "next build && next export -o dist",
```

Comment:
If you use the `next export` command, we will lose the Server Side Render Features, the Command will make Next generate an Static Site ignoring the SSR and that will affect the SEO.